### PR TITLE
Song Modules update

### PR DESCRIPTION
**Pull request by Phyco-Ninja**
removed useless flags in deezloader, repeated if/elif/else.

using spotdl as lib instead of terminal command

### CODE REVIEW

File: plugins/deezloader.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 Clogger = userge.getCLogger(__name__)
 ARL_TOKEN = os.environ.get("ARL_TOKEN")
 TEMP_PATH = 'deezdown_temp/'
-REX = re.compile(r"https?:\/\/(open\.spotify|www\.deezer)\.com\/(track|album|playlist)\/[A-Z0-9a-z]{3,}")
+REX = re.compile(r"https?:\/\/(open\.spotify|www\.deezer)\.com\/"
+                 r"(track|album|playlist)\/[A-Z0-9a-z]{3,}")
```
Comment (Krishna-Singhal) [inline plugins/deezloader.py:15]: you breaking regex 🤔🤔

File: plugins/deezloader.py [hunk 2]
```diff
@@ -118,6 +97,20 @@ async def deezload(message: Message):
...
             await message.edit("Song not Found 🚫")
             await Clogger.log(f"#ERROR\n\n{e_r}")
+        await message.delete()
+        shutil.rmtree(TEMP_PATH, ignore_errors=True)
+        return
+
+    try:
+        if 'track/' in input_link:
+            await proper_trackdl(input_link, quality, message, loader, TEMP_PATH)
+        elif 'album/' or 'playlist/' in input_link:
```
Comment (rking32) [inline plugins/deezloader.py:107]: this works ?

Comment (Krishna-Singhal) [reply plugins/deezloader.py:107]: need brackets

Comment (Phyco-Ninja) [reply plugins/deezloader.py:107]: `it works`: yes

`is it the right way`: yeah No. 😂

**Comment by rking32** [comment]
won't work this also

**Comment by Phyco-Ninja** [comment]
> won't work this also

ummm.... working fine 🤔

**Comment by Krishna-Singhal** [comment]
> won't work this also

it will 🤔🤔

**Pull request merged**